components/shared/molecules/Form/SignInForm.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Form, Formik, FormikProps } from 'formik';
+import * as Yup from 'yup';
+
+import useSignIn from 'lib/apollo/hooks/actions/useSignIn';
+
+import FormField from 'components/shared/atoms/FormField';
+import { FormFieldType } from './forms.types';
+import Button from '../../atoms/Button';
+
+const FormContentWrapper = styled.div`
+  width: 40rem;
+`;
+
+const FieldWrapper = styled.div`
+  margin-top: 1rem;
+`;
+
+const SubmitButtonWrapper = styled.div`
+  margin-top: 2rem;
+`;
+
+const initialValues = {
+  email: '',
+  password: '',
+};
+
+const SignInValidationSchema = Yup.object().shape({
+  email: Yup.string().email('Must be a valid email').max(255).required('This field is required'),
+  password: Yup.string().required('This field is required'),
+});
+
+type ValuesFromFormik = Parameters<ReturnType<typeof useSignIn>[0]>[0];
+
+const SignInFormContent = ({ isSubmitting }: FormikProps<ValuesFromFormik>) => (
+  <FormContentWrapper>
+    <Form>
+      <FieldWrapper>
+        <FormField name="email" type={FormFieldType.email} label="Email" />
+      </FieldWrapper>
+      <FieldWrapper>
+        <FormField name="password" type={FormFieldType.password} label="Password" />
+      </FieldWrapper>
+      <SubmitButtonWrapper>
+        <Button type={FormFieldType.submit} testID="submit-button" disabled={isSubmitting}>
```

Comment:
At first i've tried to use `SubmitButton` as we had before. But there is a problems: 
- if we wanna use `SubmitButton` as a separate component, we should styling it. It possible to pass attr `as` like
`<SubmitButton as={SomeStyledInputComponent} />`. But the problem is passing props such as 'testID' and 'disabled' to our `SomeStyledInputComponent`, because in this case we should use something like this:
`<SubmitButton as={(props: SubmitButtonProps) => <Button {...props} testID={testID} disabled={disabled} />}` 
But this approach negatively affects the performance. 
- And also we get the problem with `OnBlur` explaining in [this](https://github.com/jaredpalmer/formik/issues/1796) issue.

So I've decided to leave old `SubmitButton` as `Field` in `LoginForm` and use simple `Button` in `SignInForm`